launcher/ui/MainWindow.cpp

```diff
@@ -1290,6 +1301,38 @@ void MainWindow::updateToolsMenu()
     ui->actionLaunchInstance->setMenu(launchMenu);
 }
 
+void MainWindow::updateThemeMenu()
+{
+    QMenu *themeMenu = ui->actionChangeTheme->menu();
+
+    if (themeMenu) {
+        themeMenu->clear();
+    } else {
+        themeMenu = new QMenu(this);
+    }
+
+    auto themes = APPLICATION->getValidApplicationThemes();
+
+    QActionGroup* ThemesGroup = new QActionGroup( this );
```

Comment:
```suggestion
    QActionGroup* themesGroup = new QActionGroup( this );
```

(also update the usage below)